src/datachain/lib/file.py

```diff
@@ -447,13 +451,60 @@ class ImageFile(File):
     def read(self):
         """Returns `PIL.Image.Image` object."""
         fobj = super().read()
-        return Image.open(BytesIO(fobj))
+        return PilImage.open(BytesIO(fobj))
 
     def save(self, destination: str):
         """Writes it's content to destination"""
         self.read().save(destination)
 
 
+class Image(DataModel):
```

Comment:
why do we need this separate model?

Reply:
Same as for video info (`Video` model). I can remove it from this PR 🤔

Reply:
it's just a bit weird that we have ImageFile and Image (that contains only some basic metadata) 🤔

Reply:
It was `VideoMeta` (and `ImageMeta`) before, but Dmitry was asked to rename these models [here](https://github.com/iterative/datachain/pull/814#pullrequestreview-2547435306). I agree having `Video` (`Image`) model with just meta looks odd. I think you're right and we should inherit this model from `VideoFile` (`ImageFile`) to extend files with meta, than it will make sense. If no, what do you think about `VideoInfo` (and `ImageInfo`)?